task/github.py

```diff
@@ -286,15 +286,15 @@ def get(self, resource=None):
             self.cache.write(qualified, response)
             return json.loads(response['data'] or "null")
 
-    def post(self, resource, data, accept=[]):
+    def post(self, resource, data, accept=()):
```

Comment:
And again, this is a string list, not a tuple.. What does ruff complain about here?